packages/builder/src/builder.js

```diff
@@ -44,7 +44,7 @@ export default class Builder {
       restart: null
     }
 
-    this.supportedExtensions = ['vue', 'js', 'ts', 'tsx']
+    this.supportedExtensions = ['vue', 'js', 'ts', 'tsx', ...(this.options.build.customSupportedExtensions || [])]
```

Comment:
Can we simplify the option to something like `build.additionalExtensions`?

Reply:
@pi0 Sure, I had difficulties to find a proper name but your proposal is perfect :)
I'll add it in default config `packages/config/src/config/build.js` with empty array default value and will update snapshots.